Documentation/ApiOverview/Fal/Administration/Storages.rst

```diff
@@ -53,3 +53,33 @@ Is online?
         within your web root or accessible via a third-party storage service
         which is publicly available. The files will still be available to anyone
         who knows the path to the file.
+
+    ..  versionchanged:: 11.5.35/12.4.11
+
+    Assuming that a web project is located in the directory
+    :file:`/var/www/example.org/` (the "project root path" for Composer-based
+    projects) and the publicly accessible directory is located at
+    :file:`/var/www/example.org/public/` (the "public root path"), accessing
```

Comment:
I often refer to this as "DocumentRoot" / "document root" to make people understand that this is the webserver-configuration related setting (but a bit above this paragraph it mentions "web root", so that's what I used) - what do you think?

```suggestion
    :file:`/var/www/example.org/public/` (the "public root path" or "web root"), accessing
```

Reply:
Hmm, I just took this out of the changelog. "web root" sounds better. Yes.